README.md

```diff
@@ -18,6 +18,8 @@ The action is compatible with Windows, Linux, and OSX runners.  Windows self-hos
 * **hidden_files:**         List of hidden files in format '( "path1" "path2" etc.. )'
 * **sync_attributes:**      Set to "true" to sync attributes from source files.
 * **executable_files:**     List of executable files in format '( "path1" "path2" etc.. )'
+* **included_files:**       List of include file path masks, format 'path/file1;path/*mask2;path3'
+* **excluded_files:**       List of exclude file path masks, format 'path/file1;path/*mask2;path3'
```

Comment:
Why are the delimiters inconsistent for included/excluded inconsistent w/ hidden_files, executable_files, etc?

Reply:
The deploy scripts have inconsistent methods for multiple include/exclude files vs multiple hidden/executable.  Deploy scripts use a single-string with ";" delimiters method for include/exclude, so I was just passing that along.  I suppose I could use the same format as hidden, and then build the include/exclude string in the script..?

Reply:
OK, changed so they use the same format.

Reply:
The change uses spaces as the delimiter. I think semi colon would've been better. Plus you've got an issue now where spaces in the include/exclude won't work.